Initial commit

SLebedzeu opened the pull request:
This is an initial commit to check if the framework is used correctly.

SLebedzeu committed:
Initial commit

DiamondDragon commented on StoryLine.Selenium.Runner/SimpleAdviceInvestment/Actions/GoToLoginPage.cs:
```diff
@@ -0,0 +1,31 @@
+
+namespace StoryLine.Selenium.Runner.SimpleAdviceInvestment.Actions
+{
+    class GoToLoginPage : IActionBuilder, IAction
```
@SLebedzeu i would suggest to put `public` keyword here.

I think it would be logical include login actions too. Navigation in majority of cases is followed by submitting valid credentials. So it would rather implement action `LoginToPfp` (e.g. combine with action below).

DiamondDragon reviewed (commented):
PR looks good. I think we can move forward.

SLebedzeu committed:
Make LoginToPFP class public.

Combined login actions into single. Changed locator for Login button to avoid exception.

SLebedzeu committed:
Combined login actions into single. Changed locator for Login button to avoid exception.

SLebedzeu commented:
@DiamondDragon , proposed changes have been made.
